python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -273,11 +273,28 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         doc="Mask plane to use to mark pixels with negative variance, if `maskNegativeVariance` is True.",
         default="BAD",
     )
+    doSaturation = pexConfig.Field(
+        dtype=bool,
+        doc="Mask saturated pixels? NB: this is totally independent of the"
+        " interpolation option - this is ONLY setting the bits in the mask."
+        " To have them interpolated make sure doSaturationInterpolation=True",
+        default=True,
+    )
+    saturation = pexConfig.Field(
+        dtype=float,
+        doc="The saturation level to use if no Detector is present in the Exposure (ignored if NaN)",
+        default=float("NaN"),
+    )
```

Comment:
The whole way that saturation is used/overridden in the current isr task is a mess, and I'm wondering if we can figure out how to do something better before this goes live.  We should discuss.